### ISSUE TITLE
Mega menu links on mobile don't have acceptable contrast for accessibility

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1579]
Looking at www.colorado.edu, the mega menu links, when viewed on mobile, are blue on black and do not have sufficient contrast:

<img width="569" alt="Image" src="https://github.com/user-attachments/assets/beaa371a-b8c0-45de-a7c8-5733dc916774" />

**Comment by jnicholCU** [comment]
I'm going to add this to all the tickets I think it affects:
In `10.4.1` there was an issue found with CSS order being arbitrarily weighted and a fix has been merged as of last week. Hopefully that will be rolled out soon. In the meantime I will see what I can do in re-weighting our libraries.
https://www.drupal.org/project/bs_lib/issues/3490049
